### PR TITLE
docs: gh feedback link

### PR DESCRIPTION
**Pull request by baskaryan**

### CODE REVIEW

**Review by ccurme — APPROVED** [review]

File: docs/src/theme/Feedback.js [hunk 1]
```diff
@@ -240,6 +240,7 @@ export default function Feedback() {
           </div>
         </>
       )}
+      <h4>You can leave detailed feedback <a href="https://github.com/langchain-ai/langchain/issues/new?assignees=&labels=03+-+Documentation&projects=&template=documentation.yml&title=DOC%3A+%3CPlease+write+a+comprehensive+title+after+the+%27DOC%3A+%27+prefix%3E">on GitHub</a>.</h4>
```
Comment (eyurtsev) [inline docs/src/theme/Feedback.js:243]: do we have a way to add a URL to the given page?

Comment (baskaryan) [reply docs/src/theme/Feedback.js:243]: im sure someone with greater js abilities than i does have a way, cc @bracesproul

Comment (bracesproul) [reply docs/src/theme/Feedback.js:243]: Yes this is possible, can get on it today

Comment (baskaryan) [reply docs/src/theme/Feedback.js]: i tried and i failed 🌝

**Review by bracesproul — APPROVED** [review]
you were so close @baskaryan ! Just forgot to use ` instead of " for template string

**Pull request merged**